arangod/VocBase/LogicalCollection.h

```diff
@@ -105,7 +105,7 @@ class LogicalCollection : public LogicalDataSource {
   };
 
   /// @brief the category representing a logical collection
-  static Category const& category() noexcept;
```

Comment:
it has to remain `static` as member is already present in `LogicalDataSource`

Reply:
I don't think so. It is declared as pure virtual in `LogicalDataSource`.